x/uibc/errors.go

```diff
@@ -0,0 +1,14 @@
+package uibc
+
+import (
+	sdkerrors "github.com/cosmos/cosmos-sdk/types/errors"
+)
+
+var (
+	ErrIBCPauseStatus     = sdkerrors.Register(ModuleName, 1, "invalid ibc pause status")
+	ErrQuotaExceeded      = sdkerrors.Register(ModuleName, 2, "quota transfer exceeded")
+	ErrNoQuotaForIBCDenom = sdkerrors.Register(ModuleName, 3, "no quota for ibc denom")
+	ErrInvalidIBCDenom    = sdkerrors.Register(ModuleName, 4, "invalid ibc denom")
+	ErrInvalidQuota       = sdkerrors.Register(ModuleName, 5, "invalid quota")
+	ErrInvalidAmount      = sdkerrors.Register(ModuleName, 6, "invalid transfer amount")
```

Comment:
Use `status.Errorf(codes.InvalidArgument, "%v", ...)` or `sdk/errors.ErrInvalidRequest.Wrap(...)`

Reply:
and remove the not needed errors here

Reply:
ok